.eslintrc.json

```diff
@@ -57,6 +57,7 @@
         "$": false,
 
         "window": false,
+        "document": false,
```

Comment:
Sorry for the delay--I've been busy with moving, although wanted to finish and reply. That said,, yes, I can use jQuery here. Good point. And thanks for clarifying. This is my first pull request; let alone open source patch. I wasn't sure if I should use jQuery as little as possible as noobish as that sounds. This turned out to be a simple fix.  :)